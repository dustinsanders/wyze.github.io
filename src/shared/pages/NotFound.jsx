// @flow

/**
 * Credit for ghost and shadow: http://codepen.io/helenvholmes/pen/pecdI
 */

import '../styles/global'
import { GhostFloating, Shadow } from '../components/svgs'
import type { Style, VNode } from '../../types'
import { centered } from '../styles'
import { css } from 'glamor'
import H1 from '../components/H1'
import preact from 'preact'

const shadow: Style = css({
  marginTop: '1em',
  width: '8em',
})

const NotFound = (): VNode => (
  <div {...centered}>
    <GhostFloating />
    <div {...shadow}>
      <Shadow />
    </div>

    <H1>Whoops!</H1>
    <div>
      <div>We couldn&apos;t find the page you</div>
      <div>were looking for.</div>
    </div>
  </div>
)

export default NotFound
