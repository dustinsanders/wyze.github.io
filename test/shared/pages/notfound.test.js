// @flow

import NotFound from '../../../src/shared/pages/NotFound'
import snapshot from '../../helpers/snapshot'

describe('<NotFound />', () => {
  it('renders', snapshot.bind(null, NotFound))
})
