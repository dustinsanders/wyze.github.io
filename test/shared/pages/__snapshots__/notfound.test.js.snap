// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders 1`] = `
"<div data-css-jgfdvp>
  <GhostFloating></GhostFloating>
  <div data-css-1ka2kui>
    <Shadow></Shadow>
  </div>
  <H1>Whoops!</H1>
  <div>
    <div>We couldn't find the page you</div>
    <div>were looking for.</div>
  </div>
</div>"
`;
